mat/cholesky.go

```diff
@@ -340,6 +340,70 @@ func (c *Cholesky) Scale(f float64, orig *Cholesky) {
 	c.cond = orig.cond // Scaling by a positive constant does not change the condition number.
 }
 
+// AppendOneSym computes the Cholesky decomposition of the original matrix A
```

Comment:
I'm not enthralled by the doc comment; it reads as though the `orig` value is updated, but it is not.

My suggestion (still not good - I usually defer to @vladimir-ch for documentation clarity):

```
// ExtendVecSym computes the Cholesky decomposition of the matrix A, whose
// Cholesky decomposition is in a, with extended with an n×1 vector symmetrically
// according to
//  [A  v]
//  [v' k]
// where k = v[n-1] and v = v[:n-1]. The result is stored into the receiver.
// In order for the updated matrix to be positive definite, it must be the
// case that n > v' A^-1 v. If this condition does not hold then ExtendVecSym
// will return false and the receiver will not be updated.
//
// ExtendVecSym will panic if len(v) != a.Size()+1 or if a does not contain
// a valid decomposition.
```

This requires that the notation in the comments below changes, but that notation is already potentially confusing since the A/a typography suggests a relationship between them that does not exist.

Reply:
ExtendedVecSym is a much better name, thanks.